Solutions/Problem_Statement_59_Solution.py

```diff
@@ -0,0 +1,40 @@
+"""
+There are three pegs labelled A, B and C. Four disks are placed on peg A. The bottom most disk is
+largest, and disk go on decreasing in size with topmost disk being smallest. The objective of the
+game is to move the disks from peg A to peg C using peg B as an auxiliary peg.
+The rules of the game are:
+
+Only one disc may be moved at a time, and it must be the top disk on one of the pegs.
+A larger disk should never be placed on the top of a smaller disk.
+Write a program to print out the sequence in which the disk should be moved such that all the disks
+on peg A are finally transferred to peg C.
+-----------------------------------------------------------------------------------------------
+ Tower of Hanoi problem
+ A= from/source peg, B= aux peg, C= to/ destination peg
+"""
+def TOH(n,A,B,C):
+    if n>0:
+        TOH(n-1, A, C, B)
+        print(f"Move disk from {A} to {C}.")
+        TOH(n-1,B, A, C)
+ 
+no_of_disks= 4
+TOH(no_of_disks,'A','B','C')    
+
+#---------------------------------------------------------------------------------------------
+# OUTPUT:
+# Move disk from A to B.
+# Move disk from A to C.
```

Comment:
Not getting same output as given here! Pls Re-try the question

Reply:
expected output is: moved disk [No.] from [peg name] to [peg name]. No. is not being displayed

Reply:
okay I will add it. The expected output was not mentioned in the problem statement.

Reply:
Done. Please check now

Reply:
Yes, The output is correct. And displaying the disk number as well. Sorry for inconvenience.